apps/ledger-live-desktop/src/renderer/families/tron/AccountBalanceSummaryFooter.tsx

```diff
@@ -63,14 +63,32 @@ const AccountBalanceSummaryFooter = ({ account }: Props) => {
     locale,
   };
   const energy = tronResources.energy;
-  const bandwidthAmount = tronResources.frozen.bandwidth?.amount;
-  const energyAmount = tronResources.frozen.energy?.amount;
+  const frozenBandwidthAmount = tronResources.frozen.bandwidth?.amount;
+  const frozenEnergyAmount = tronResources.frozen.energy?.amount;
+  const delegatedFrozenBandwidthAmount = tronResources.delegatedFrozen.bandwidth?.amount;
+  const delegatedFrozenEnergyAmount = tronResources.delegatedFrozen.energy?.amount;
+  const unfrozenBandwidthAmount = tronResources.unFrozen.bandwidth
+    ? tronResources.unFrozen.bandwidth.reduce((accum, cur) => {
+        return accum.plus(cur.amount);
+      }, new BigNumber(0))
+    : new BigNumber(0);
+  const unfrozenEnergyAmount = tronResources.unFrozen.energy
+    ? tronResources.unFrozen.energy.reduce((accum, cur) => {
+        return accum.plus(cur.amount);
+      }, new BigNumber(0))
+    : new BigNumber(0);
+
   const { freeUsed, freeLimit, gainedUsed, gainedLimit } = tronResources.bandwidth;
 
   const spendableBalance = formatCurrencyUnit(account.unit, account.spendableBalance, formatConfig);
   const frozenAmount = formatCurrencyUnit(
     account.unit,
-    BigNumber(bandwidthAmount || 0).plus(BigNumber(energyAmount || 0)),
+    BigNumber(frozenBandwidthAmount || 0)
+      .plus(BigNumber(frozenEnergyAmount || 0))
+      .plus(BigNumber(delegatedFrozenBandwidthAmount || 0))
+      .plus(BigNumber(delegatedFrozenEnergyAmount || 0))
+      .plus(BigNumber(unfrozenBandwidthAmount))
+      .plus(BigNumber(unfrozenEnergyAmount)),
```

Comment:
Just for my understanding, `unfrozenBandwidthAmount` and `unfrozenEnergyAmount` are actually pat of the frozen amount because they are not available yet, until the unfreezing period is complete, is this correct?

Reply:
Correct. The unfrozen balances are still technically frozen until the user withdraws the funds. However, the Tron Power and resources will be released immediately upon calling `Unfreeze`.